File: JavaScript/Account.js
class Account{
    constructor(name, document, email, verified){
        this.id
        this.name = name
        this.document = document
        this.email = email
        this.password = this.password
        this.verified = verified
    }

    printDataAccount = () => {
        console.log(`Name: ${this.name} Document: ${this.document} Email: ${this.email} Verified: ${this.verified}`)
    }
}

export default Account

File: Java/Account.java
package Java;

public class Account {
    Integer id;
    public String name;
    String document;
    String email;
    String password;
    Boolean verified;

    public Account(String name, String document, String email, String password, Boolean verified){
        this.name = name;
        this.document = document;
        this.email = email;
        this.password = password;
        this.verified = verified;
    }

    public void printAccount(){
        System.out.println("Name: " + name + " Document: " + document + " Email: " + email + " Verified: " + verified);
    }
}
